### PR TITLE
added select component

### PR DESCRIPTION
**Pull request by rohitgupta274509**
Sample Output:-

image

Please review and approve it.

### CODE REVIEW

File: application/backend/src/main/resources/apps/typerefinery/components/forms/fields/select/clientlibs/behaviour.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+window.Typerefinery = window.Typerefinery || {};
+window.Typerefinery.Components = Typerefinery.Components || {};
+window.Typerefinery.Components.Forms = Typerefinery.Components.Forms || {};
+window.Typerefinery.Components.Forms.Select = Typerefinery.Components.Forms.Select || {};
+
+;(function (ns, document) {
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/forms/fields/select/clientlibs/behaviour.js:6]: As of now I don't think we need any clientlibs for the select component.

File: application/backend/src/main/resources/apps/typerefinery/components/forms/fields/select/variant.html [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
-        <div class="dropdown-label">
-            <label>${model.label}</label>
-        </div>
-        <p-dropdown class="dropdown-placeholder" placeholder="${model.placeholder}" list="${model.list}">
-        </p-dropdown>
+    <div 
+        class="${model.componentClassNames}" 
+        id="${model.id}" 
+        data-model="${model.jsonString}">
+        <select class="${model.variantClassNames}">
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/forms/fields/select/variant.html:6]: format the code.

**Review by Harikarthyk — CHANGES_REQUESTED** [review]

File: application/backend/src/main/resources/apps/typerefinery/components/forms/fields/select/clientlibs/behaviour.js (hunk 1 above)
Comment (rohitgupta274509) [reply application/backend/src/main/resources/apps/typerefinery/components/forms/fields/select/clientlibs/behaviour.js]: Deleted from code.

File: application/backend/src/main/resources/apps/typerefinery/components/forms/fields/select/variant.html (hunk 2 above)
Comment (rohitgupta274509) [reply application/backend/src/main/resources/apps/typerefinery/components/forms/fields/select/variant.html:6]: Done.

**Review by wildone — APPROVED** [review]
can we add some tests as well?

**Pull request merged**